pkg/fleet/installer/setup/common/setup.go

```diff
@@ -114,23 +113,14 @@ func (s *Setup) Run() (err error) {
 	for _, p := range packages {
 		s.Out.WriteString(fmt.Sprintf("  - %s / %s\n", p.name, p.version))
 	}
-	err = installinfo.WriteInstallInfo(fmt.Sprintf("install-%s.sh", s.flavor))
+	// TODO(WINA-1341): This is being overwritten by the MSI on Windows
+	err = installinfo.WriteInstallInfo(fmt.Sprintf("fleet-setup-%s", s.flavor))
```

Comment:
Is there a specific reason why the template changed? This script is used in non-fleet management scenarios (mostly for DJM), so having `fleet-setup-databricks` (for instance) reported as the agent install method feels a bit weird

Reply:
@BaptisteFoy woops I forgot to DM you about that

@julien-lebot typo, should be  [WINA-1431](https://datadoghq.atlassian.net/browse/WINA-1431) 🙇